pandas/core/frame.py

```diff
@@ -7565,6 +7567,16 @@ def nlargest(
         Maldives      434000     4520      MV
         Brunei        434000    12128      BN
 
+        However, ``nlargest`` does not keep ``n`` distinct largest elements:
+        
+        >>> df.nlargest(5, 'population', keep='all')
```

Comment:
This is a great example, can you also include a similar one in the docstring of `nsmallest`?

Reply:
@rhshadrach  done.